src/library/blas/xgemm.cc

```diff
@@ -259,7 +260,7 @@ void makeGemmKernel(
       printf("%s\n", buildLog);
       //printf("\n\nKernel String:\n\n");
       //printf("%s\n", kernelSource);
-      //FIXME: The function should be exiting at this point
+      CL_CHECK(build_err);
```

Comment:
Just for consistency with other points of failure (message and assert), so if CL_CHECK ever gets changed to something more, the change will automatically propagate there too. (This specific change is not for portability, but it has made some debugging easier.)